XenAdmin/Controls/Wlb/WlbReportView.cs

```diff
@@ -142,12 +141,6 @@ public IEnumerable<Host> Hosts
             }
         }
 
-        public bool IsCreedenceOrLater

```

Comment:
I dont't think that IsCredenceOrGreater should be removed, because we are still supporting Clearwater, which is pre-Creedence, so if there is functionality which only applies to Creedence or greater, then we should keep it that way.